gapic-generator/lib/gapic/presenters/gem_presenter.rb

```diff
@@ -183,8 +183,8 @@ def extra_files
       end
 
       def dependencies
-        deps = { "gapic-common" => "~> 0.4" }
-        deps["grpc-google-iam-v1"] = [">= 0.6.10", "< 2.0"] if iam_dependency?
+        deps = { "gapic-common" => [">= 0.4.1", "< 2.a"] }
```

Comment:
what does `2.a` exclude (or include) in this context that `2.0` would not? https://guides.rubygems.org/patterns/ suggests it having someting to do with prereleases?

Reply:
Yes, `2.a` is the "earliest" possible 2.x prerelease. For example, `2.0.rc1` would be allowed by `< 2.0`, but not by `< 2.a`.